src/main/java/com/zamoiski/error/RestExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.zamoiski.error;
+
+import com.zamoiski.entity.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionHandler {
+    @ExceptionHandler
+    public ResponseEntity<ErrorResponse> handleException(NotFoundException exc){
+        ErrorResponse errorResponse=new ErrorResponse(HttpStatus.NOT_FOUND,exc.getMessage());
+        return new ResponseEntity<>(errorResponse,HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler
+    public ResponseEntity<ErrorResponse> handleException(Exception exc){
+        ErrorResponse errorResponse=new ErrorResponse(HttpStatus.BAD_REQUEST,exc.getMessage());
+        return new ResponseEntity<>(errorResponse,HttpStatus.BAD_REQUEST);
+    }
+}
```

Comment:
NotFoundException extends RuntimeException extends Exception. What status will be return if NotFoundException exception?